src/background/helpers.js

```diff
@@ -23,6 +23,24 @@ chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
     case 'openPrivateWindowWithUrl':
       chrome.windows.create({ url: msg.url, incognito: true });
       break;
+    case 'openElementPicker':
+      chrome.scripting.executeScript(
+        {
+          injectImmediately: true,
+          world: chrome.scripting.ExecutionWorld?.ISOLATED ?? 'ISOLATED',
```

Comment:
We had this pattern elsewhere, maybe worth to share the code?

Reply:
The util file for a function with one line only used in a specific case? :P Please no.